src/components/Capture/capture.js

```diff
@@ -134,11 +134,20 @@ class Capture extends Component {
     this.onImageFileSelected(file)
   }
 
-  onWebcamError = () => {
-    this.setState({uploadFallback: true})
+  onWebcamError = (error) => {
```

Comment:
Should we not log an error? `console.error` if nothing else?

Reply:
Yes, that would be useful for devs. As part of another story we will also display the error to the user, if they have granted permission but there is an issue with the stream.

Reply:
The `react-webcam` dependency already returns an error here.